app/scicrunch_processing_common.py

```diff
@@ -32,6 +33,7 @@
     'inode/vnd.abi.scaffold+thumbnail': SCAFFOLD_THUMBNAIL,
     'inode/vnd.abi.scaffold.thumbnail+file': SCAFFOLD_THUMBNAIL,
     'inode/vnd.abi.scaffold.view+file': SCAFFOLD_VIEW_FILE,
+    'application/x.abi.context-information+json': CONTEXT_FILE,
```

Comment:
This isn't the correct mimetype it should be `application/x.vnd.abi.context-information+json`

Reply:
Thanks yeah it is currently using ` application/x.abi.context-information+json`, but yes I'll switch to the new one